src/build/usage_parser.rs

```diff
@@ -264,7 +261,6 @@ mod test {
         assert_eq!(a.about.unwrap(), "some help info");
         assert!(!a.is_set(ArgSettings::MultipleOccurrences));
         assert!(a.val_names.is_empty());
-        assert!(a.num_vals.is_none());
```

Comment:
Why was this removed?  Won't it still be `None`?

Reply:
Yes, but it's unnecessary in these unit tests because we are not touching `num_vals`. We don't want to test all the arg field defaults here.

Reply:
While I can understand the slippery slope of "what to test", I think testing cases where inference didn't occur is useful

EDIT: Nevermind, these are tests for usage, they should stay focused on usage and not be covering inference.